src/Form/RestaurantType.php

```diff
@@ -109,7 +110,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         '%entity%' => 'restaurant',
                     ],
                     'help_html' => true,
-                ));
+                ))->add('billingMethod', ChoiceType::class, [
+                    'label' => 'form.billing_method.label',
+                    'help' => 'form.billing_method.help',
+                    'choices' => [
+                        'form.billing_method.per_task' => 'per_task',
+                        'form.billing_method.total_percentage' => 'total_percentage',
```

Comment:
Missing translations + maybe we should name the choices: 
- `unit`
- `percentage`